lib/reducer.js

```diff
@@ -1,13 +1,13 @@
-'use strict';
+"use strict";
 
-const actions = ['pick', 'fixup', 'squash', 'reword', 'edit', 'drop'];
+const actions = ["pick", "fixup", "squash", "reword", "edit", "drop"];
 const [DOWN, UP] = [1, -1];
 
 function deepFreeze(obj) {
   var propNames = Object.getOwnPropertyNames(obj);
   propNames.forEach(function (name) {
     var prop = obj[name];
-    if (typeof prop === 'object' && prop !== null) {
+    if (typeof prop === "object" && prop !== null) {
```

Comment:
Thanks for the effort @real-yfprojects. There's a lot of noise in the PR like these (double-quote to single quote changes, added trailing comma etc). Could you edit it to minimize the diff? to more carefully try and separate functionality from style changes.

Reply:
I am sorry. I forgot about that side effect of running a code formatter.